### ISSUE TITLE
WebPDF not working from Jupyter Lab export menu

### ISSUE THREAD

**Issue by joelostblom** [jupyter/nbconvert#1417]
First: The new webpdf option is just **fantastic**!! THANK YOU for implementing this!

It works flawlessly for me via `jupyter nbconvert --to webpdf`, but when using the `WebPDF` option in the Jupyter Lab export menu, I get an error saying `This even loop is already running`. I tried updating my Jupyter Lab and also rebooted for good measure but to no avail.

<details><summary>Full terminal output</summary>
<pre>
[I 17:21:16.720 LabApp] Building PDF
[E 17:21:16.720 LabApp] nbconvert failed: This event loop is already running
    Traceback (most recent call last):
      File "/home/joel/miniconda3/lib/python3.7/site-packages/notebook/nbconvert/handlers.py", line 132, in get
        resources=resource_dict
      File "/home/joel/miniconda3/lib/python3.7/site-packages/nbconvert/exporters/webpdf.py", line 81, in from_notebook_node
        pdf_data = self.run_pyppeteer(html)
      File "/home/joel/miniconda3/lib/python3.7/site-packages/nbconvert/exporters/webpdf.py", line 71, in run_pyppeteer
        pdf_data = asyncio.get_event_loop().run_until_complete(main())
      File "/home/joel/miniconda3/lib/python3.7/asyncio/base_events.py", line 563, in run_until_complete
        self._check_runnung()
      File "/home/joel/miniconda3/lib/python3.7/asyncio/base_events.py", line 523, in _check_runnung
        raise RuntimeError('This event loop is already running')
    RuntimeError: This event loop is already running
[W 17:21:16.721 LabApp] 500 GET /nbconvert/webpdf/Untitled1.ipynb?download=true (127.0.0.1): nbconvert failed: This event loop is already running
[E 17:21:16.722 LabApp] {
      "Host": "localhost:8888",
      "Connection": "keep-alive",
      "Upgrade-Insecure-Requests": "1",
      "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36",
      "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9",
      "Sec-Fetch-Site": "same-origin",
      "Sec-Fetch-Mode": "navigate",
      "Sec-Fetch-Dest": "document",
      "Referer": "http://localhost:8888/lab",
      "Accept-Encoding": "gzip, deflate, br",
      "Accept-Language": "en-GB,en-US;q=0.9,en;q=0.8",
      "Cookie": "_xsrf=2|5417ade7|5622df2d8c2f3191daceb216f5b016d5|1600255670; username-localhost-8889=\"2|1:0|10:1600819422|23:username-localhost-8889|44:ZGE2ODFjZDgxNmY2NGE5ZTkyYTE2ZWQ5MDYwZjQ0NmY=|ab617d7032ac9454a16738c4f9f5d2f0a721e75c6e2683c7e4293b87d236e5f8\"; username-localhost-8888=\"2|1:0|10:1600820474|23:username-localhost-8888|44:NGI2MmE1NDNlNDAyNGMyNWI1MzNkNjI0MzljZjNiYjA=|2fe6494ee29afa41e190a4b9aa03edd8d8baa225efd2c54aa05196e8511cb08d\""
    }
[E 17:21:16.722 LabApp] 500 GET /nbconvert/webpdf/Untitled1.ipynb?download=true (127.0.0.1) 202.04ms referer=http://localhost:8888/lab
/home/joel/miniconda3/lib/python3.7/asyncio/base_events.py:1771: RuntimeWarning: coroutine 'WebPDFExporter.run_pyppeteer.<locals>.main' was never awaited
  handle = self._ready.popleft()
RuntimeWarning: Enable tracemalloc to get the object allocation traceback

</pre></details>

I installed `nbconvert` via conda, `nbconver[webpdf]` via pip, and chromium via the optional flag. I'm on Ubuntu 20.04.

```
jupyter core     : 4.6.3
jupyter-notebook : 6.1.4
qtconsole        : not installed
ipython          : 7.18.1
ipykernel        : 5.3.4
jupyter client   : 6.1.7
jupyter lab      : 2.2.8
nbconvert        : 6.0.5
ipywidgets       : not installed
nbformat         : 5.0.7
traitlets        : 5.0.4
```

![image](https://user-images.githubusercontent.com/4560057/93950426-510b3780-fcf8-11ea-98b5-8e4419970802.png)

**Comment by SylvainCorlay** [comment]
Thanks for reporting @joelostblom. I have just pushed a fix in a PR.

I plan on pushing out a new patch release very soon with this and something else.

**Comment by SylvainCorlay** [comment]
This has been fixed in 6.0.6 which was just tagged.

This version includes some other improvements to the WebPDF exporter.

**Comment by joelostblom** [comment]
Thanks for the super quick fix!
